owncloudApp/src/main/res/layout/remove_files_dialog.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:elevation="10dp"
```

Comment:
All these measures that are written raw here (except for `0dp`) should be part of the `dims.xml` resource file, or use one of the values we already have there.
In this case, is it necessary that we set the elevation? The one by default is not suitable enough for a dialog?

Reply:
I've changed it with a dimensions that was in dimensions.xml